third_party/amd/python/triton_amd.cc

```diff
@@ -161,6 +161,18 @@ void init_triton_amd(py::module &&m) {
       module->eraseNamedMetadata(openclVersion);
   });
 
+  m.def("disable_print_inline", [](llvm::Module *module) {
```

Comment:
`disable_inline_print`

Reply:
I don't get it, can you elaborate?

My idea for this naming is something like "disable function inline attribute" or "disable print inlining".
`disable_inline_print` sounds more like "disable <some variant of> print" for me.